docsrc/markdown/datafeed.md

```diff
@@ -1,4 +1,9 @@
 # Datafeed
+> The datafeed 1 service will be fully replaced by the datafeed 2 service in the future. 
+> Please consider using datafeed 2.  
+> 
+> For more information on the timeline as well as on the benefits of datafeed 2, please reach out to your Technical 
+> Account Manager or to our [developer documentation](https://docs.developers.symphony.com/building-bots-on-symphony/datafeed).
```

Comment:
Could you as well update the configuration.md file at lines 158-160 to say the default is now DF v2?

Reply:
Done here https://github.com/finos/symphony-bdk-python/pull/270/commits/9f6da7f27ede3f93270e8c284e7f9a7441b51506